presto-docs/src/main/sphinx/release/release-0.173.rst

```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.173
+=============
+
+General Changes
+---------------
+
+* Fix issue where selective aggregates for ``count`` with a constant argument were being ignored.
```

Comment:
Sure, but FWIW, that's the name of the feature (just like "common table expressions" vs WITH, or window functions vs OVER).